pom.xml

```diff
@@ -109,6 +110,13 @@
           <version>${grpc.version}</version>
           <scope>test</scope>
         </dependency>
+        <dependency>
+          <groupId>io.netty</groupId>
+          <artifactId>netty-tcnative-boringssl-static</artifactId>
+          <version>${netty-tcnative.version}</version>
+          <classifier>${os.detected.classifier}</classifier>
+          <scope>test</scope>
```

Comment:
I think I know what's going on. `Netty transport` used by jetcd's grpc needs a mechanism to do TLS. The `netty-tcnative-boringssl-static` includes `BoringSSL` where grpc can use it to establish secure connection. On the other hand, user can also set up `Openssl` locally and use `netty-tcnative` to dynamically link to the local `Openssl` binary to establish secured connection. Is that what's going on?

I think we might need to add doc on how to properly tls on jetcd.  This link is helpful https://github.com/grpc/grpc-java/blob/master/SECURITY.md